pkg/build/options.go

```diff
@@ -89,9 +89,9 @@ func WithConfig(buildConfigs map[string]Config) Option {
 //
 // platform = <os>[/<arch>[/<variant>]]
 // allowed = all | platform[,platform]*
-func WithPlatforms(platforms string) Option {
+func WithPlatforms(platforms []string) Option {
```

Comment:
@jonjohnsonjr had the good idea that this change could be `WithPlatforms(platforms string...)` and not break existing users. (Thanks Jon!)

If you feel like doing this change in this PR that'd be great, if not I can do it in a followup before we release this.

Otherwise, lgtm!

Reply:
Ah yes! I felt silly I didn't think of it in the first place. Fixing...

Reply:
🤔 actually, using `...string` would not break compile-time for existing users, but it would break the behavior, unless we do some sort of `strings.Spilt` here.

Do we want to add that logic as well? Something like:

```go
func WithPlatforms(platforms ...string) Option {
  if len(platforms) == 1 {
    platforms = strings.Split(platforms, ",")
  }
  ...
}
```

There is an itch on me that this code is waiting to be deleted, but happy to make changes if maintainers are ok with maintaining it 😄

Reply:
Yeah let's add the behavior you describe, for folks using `WithPlatforms("linux/amd64,linux/arm64")`, and document that this behavior will change in a future release. This is a fairly small concession we should make to existing library users.